src/style/utils/merge-deep.spec.js

```diff
@@ -41,4 +41,9 @@ describe('mergeDeep', () => {
   it('merges the objects but does not create a new key when not passed an object', () => {
     expect(mergeDeep(base, undefined)).toEqual(base);
   });
+
+  it('is not mutative', () => {
+    mergeDeep(base, toMerge);
+    expect(base).toEqual(base);
```

Comment:
This is always going to pass, its the same as `expect(true).toEqual(true);` - you need to assert against some actual properties of the original object to conclude they haven't changed after the merge.